samples/bluetooth/mesh/chat/prj.conf

```diff
@@ -54,7 +54,6 @@ CONFIG_BT_MESH_DK_PROV=y
 # Enable Bluetooth mesh models debug logs
 CONFIG_BT_DEBUG_LOG=y
 CONFIG_BT_MESH_DEBUG=y
-CONFIG_BT_MESH_DEBUG_MODEL=y
```

Comment:
What is the reason for disabling model debugging here but leaving mesh stack debugging?  App demonstrates vendor model implementation.